src/narrativelog/routers/edit_message.py

```diff
@@ -30,6 +30,26 @@ async def edit_message(
         description="Tags describing the message, as space-separated words. "
         "If specified, replaces the existing set of tags. " + TAG_DESCRIPTION,
     ),
+    systems: None
+    | list[str] = fastapi.Body(
+        default=None,
+        description="Zero or more systems to which the message applied. "
+        "If specified, replaces all existing systems.",
+    ),
+    subsystems: None
+    | list[str] = fastapi.Body(
+        default=None,
+        description="Zero or more subsystems to which the message applies. "
+        "If specified, replaces all existing subsystems.",
+    ),
+    cscs: None
+    | list[str] = fastapi.Body(
+        default=None,
+        description="Zero or more CSCs to which the message applies. "
+        "Each entry should be in the form 'name' or 'name:index', "
+        "where 'name' is the SAL component name and 'index' is the SAL index. "
+        "If specified, replaces all existing cscs.",
```

Comment:
`cscs` -> `CSCs`

Reply:
Good point. I decided to change all of the "If specified..." doc lines in the edit_message docs with:
```
"If specified, replaces all existing entries."
```
which I hope is clear and avoids the dichotomy between using CSCs (any case) to refer to CSC names and the `cscs` column.